README.md

```
@@ -1,3 +1,4 @@
+##[![SonarQube Cloud](https://sonarcloud.io/images/project_badges/sonarcloud-dark.svg)](https://sonarcloud.io/summary/new_code?id=0hmware_ogz-source)
```

markdownlint-cli2 (0.17.2)

No space after hash on atx style heading

(MD018, no-missing-space-atx)